src/app/api/api_v1/endpoints/cameras.py

```diff
@@ -27,6 +27,8 @@ async def register_camera(
     token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN, UserRole.AGENT]),
 ) -> Camera:
     telemetry_client.capture(token_payload.sub, event="cameras-create", properties={"device_login": payload.name})
+    if token_payload.organization_id != payload.organization_id and UserRole.ADMIN not in token_payload.scopes:
```

Comment:
Perhaps a suggestion:
- make org_ID optional in the payload
- if it's none, defaults to the org ID of the authenticated user
- if it's not none, it forwards it if this is an admin, but throws an error if it's an agent "Your role is not authorized to override the field "organization_id""

Reply:
I really don't like that org_id could be None, for me it's a security hole